lib/plugins/acl/action.php

```diff
@@ -1,24 +1,27 @@
 <?php
+
+use dokuwiki\Extension\ActionPlugin;
+use dokuwiki\Extension\EventHandler;
+use dokuwiki\Extension\Event;
+
 /**
  * AJAX call handler for ACL plugin
  *
  * @license    GPL 2 (http://www.gnu.org/licenses/gpl.html)
  * @author     Andreas Gohr <andi@splitbrain.org>
  */
-
 /**
  * Register handler
  */
-class action_plugin_acl extends DokuWiki_Action_Plugin
+class action_plugin_acl extends ActionPlugin
 {
-
     /**
      * Registers a callback function for a given event
      *
      * @param Doku_Event_Handler $controller DokuWiki's event controller object
      * @return void
      */
-    public function register(Doku_Event_Handler $controller)
+    public function register(EventHandler $controller)
```

Comment:
changes like this don't update the phpdoc

Reply:
In one case phpdocs was updated, might be a manual action of @splitbrain or because it was prefixed with a`\`. This concerns a lot of the changes in 8553d24d33ab5f260c6e19959de764dd8472d438

Reply:
I fixed a bunch of them. If you see any more please fix them